README.md

```diff
@@ -13,3 +13,65 @@ elements in ApertureDB at the object level.
 * NotebookHelpers.py provides helpers to show images/bounding boxes on Jupyter Notebooks
 
 For more information, visit https://aperturedata.io
+
+
+## Object Mapper API
+This API allows the user to interact with the database with the idea of objects that can be Created, Read, Updated and Deleted through an intutive interface involving these objects.
+
+It supports the following types of Objects in the Database as Python objects:
+| Object | Dcumentation |
+| ----- | -----|
+| Blob||
+| BoundingBox || 
+| Descriptor||
+| DescriptorSet||
+| Entity||
+| Image||
+| Video||
+
+
+### Examples
+**Create an Image**
```

Comment:
this is a great start, but what we would like is two separate sets of documentation: 
1) docstrings or similar, to let users understand the API. Ideally, we would like to generate an html documentation (like this: https://pytorch.org/docs/stable/torch.html (which actually uses read-the-docs))
2) Example documentation, for users to see some use-cases. For the examples, you can consider using read-the-docs/sphinx as we are using for documentation ApertureDB API